app/assets/javascripts/3dbio_viewer/src/webapp/components/network/IncludeNeighborsCheckbox.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import i18n from "../../utils/i18n";
+import "./Network.css";
+
+interface IncludeNeighborsCheckboxProps {
+    checkedValue: boolean;
+    onCheckboxChange: () => void;
+}
+
+const IncludeNeighborsCheckbox: React.FC<IncludeNeighborsCheckboxProps> = React.memo(props => {
+    const { checkedValue, onCheckboxChange } = props;
+    return (
+        <div className="include-neighbors-checkbox">
+            <input
+                type="checkbox"
+                className="include-neighbors-checkbox"
+                checked={checkedValue}
+                onChange={() => onCheckboxChange()}
```

Comment:
Better: `onChange={onCheckboxChange}`, which also prevents a new arrow function to be created and the prop remains immutable (important for pure components performance)